go.mod

```diff
@@ -29,3 +29,5 @@ require (
 	knative.dev/test-infra v0.0.0-20190730202142-17f2331e80ad
 	sigs.k8s.io/yaml v1.1.0
 )
+
+replace github.com/spf13/cobra => github.com/chmouel/cobra v0.0.0-20190820110723-8f09bb39b20d
```

Comment:
How can we track that we remove the temporary replacement when the next cobra release is out ?

otherwise I think its ok for me to include this bug-fix release here (it's about escaping `[` in the help message, right ?)

But we should decide first on #439 which direction we want to go.

Reply:
We can't track that, we can open a ticket though in client to switch to cobra and we could cross check part of triaging client issues.

> otherwise I think its ok for me to include this bug-fix release here (it's about escaping [ in the help message, right ?)

Yes.

About the comment on #439 

> if we don't use [ in our help messages)

There are some flags we inherit for machine readable output like `--template` whose help text we don't control. :-/

About using cobra from forked bug fix branch, I'd have deferred it if cobra PR turn around is quicker, apparently the PR from Chmouel is open since a while and this ended up even tkn using the fork of cobra.